controller/src/main/java/org/jboss/as/controller/services/path/PathInfoHandler.java

```diff
@@ -100,6 +100,20 @@
                         .setDefaultValue(new ModelNode(MeasurementUnit.BYTES.getName()))
                         .build();
 
+    private static DateTimeFormatter getDateFormat() {
+        if (DATE_FORMAT == null) {
+            DATE_FORMAT = new DateTimeFormatterBuilder().appendInstant().appendZoneId().toFormatter(Locale.ENGLISH);
+        }
+        return DATE_FORMAT;
+    }
+
+    private static ZoneId getZoneId() {
+        if (ZONE_ID == null) {
+            ZONE_ID = ZoneId.of(Calendar.getInstance().getTimeZone().getID());
```

Comment:
Can we just use `ZoneId.systemDefault()` here?

Reply:
I'll add a commit to change it to that.